app/assets/stylesheets/active_admin.scss

```diff
@@ -17,3 +17,4 @@
 @import "active_admin/mixins";
 @import "active_material";
 @import "admin/**/*";
+@import "application/components/utility_nav"
```

Comment:
The admin directory is where styles for active admin should go. We use application only for user facing styles.

Reply:
okey dokey